CHANGELOG.md

```diff
@@ -36,16 +36,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Removed dataType/version from api granule schema
   - Added `@cumulus/api/endpoints/granules` unit to cover duration overwrite
     logic for PUT/PATCH endpoint
+
 ### Changed
 
--**CUMULUS-3100**
```

Comment:
Hmm this actually might be unreleased. By the release notes anyway it looks like this was _not_ released in 13.4: https://github.com/nasa/cumulus/releases/tag/v13.4.0

Reply:
Hmm, that's not good.   👀